services/accounting/controllers/callback.ts

```diff
@@ -1,10 +1,69 @@
 const log = require("loglevel");
 import asyncHandler from "express-async-handler";
 import prisma from "@prismalocal/client";
+import { validate } from "uuid";
+import core = require("express-serve-static-core");
+import { ZBDKeysendCallback } from "@library/zbd/interfaces";
+import { buildAmpTx } from "@library/amp";
+import db from "@library/db";
+import { KeysendMetadata } from "@library/keysend";
 
-const processIncomingKeysend = asyncHandler(async (req, res, next) => {
-  // TODO - process an incoming keysend payement
-  res.status(200);
+const BLIP0010 = "7629169";
+const NOT_SURE_INTERNAL_KEY = "16180339";
```

Comment:
This is the custom key Wavlake uses in RSS feeds for clients to attach the content id value to
```suggestion
const WAVLAKE_CUSTOM_KEY = "16180339";
```